src/setup.py

```diff
@@ -18,27 +18,22 @@
 from setuptools import setup, find_packages
 
 setup(
-    name='katello-agent',
-    version='3.5.7',
-    description='Katello Agent',
-    author='Jeff Ortel',
-    author_email='jortel@redhat.com',
+    name='katello-host-tools',
+    version='3.5.8',
```

Comment:
I think I'd make this at least 3.6.0. Renaming it may have impact on packaging as well.
```suggestion
    version='3.6.0',
```

Reply:
Bumped to 4.0 - seemed appropriate